gateway-ha/pom.xml

```diff
@@ -56,14 +56,14 @@
           <version>2.10.1</version>
         </dependency>
         <dependency>
-            <groupId>com.lyft.data</groupId>
+            <groupId>io.trino.gateway</groupId>
             <artifactId>baseapp</artifactId>
-            <version>${project.parent.version}</version>
+            <version>1.9.5</version>
```

Comment:
Why hardcode the version here .. this seems wrong .. if anything you can set the version in dependencyManagement in the parent.

Reply:
yes, we can just take the parent's version

Reply:
ok!